.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -40,3 +42,11 @@ I tried this:
 I expected to see this happen: [explanation]
 
 Instead, this happened: [explanation]
+
+**Commands**
+
+Introduce the exact sequence of command line instructions executed so the team can reproduce the problem.
```

Comment:
It's better to be specific - a lot of people put commands and logs in their own words.
```suggestion
Copy and paste the exact commands you used, so the team can try to reproduce the issue.
```